.github/actions/pr_notifier/pr_notifier.py

```diff
@@ -0,0 +1,171 @@
+# Script for collecting PRs in need of review, and informing reviewers via
+# slack.
+#
+# By default this runs in "developer mode" which means that it collects PRs
+# associated with reviewers and API reviewers, and spits them out (badly
+# formatted) to the command line.
+#
+# .github/workflows/pr_notifier.yml runs the script with --cron_job
+# which instead sends the collected PRs to the various slack channels.
+#
+# NOTE: Slack IDs can be found in the user's full profile from within Slack.
+
+from __future__ import print_function
+
+import argparse
+import datetime
+import os
+import sys
+
+import github
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+
+REVIEWERS = {
+    'alyssawilk': 'U78RP48V9',
+    'junr03': 'U79K0Q431',
+    'RyanTheOptimist': 'U01SW3JC8GP',
+    'goaway': 'U7TDPD3L2',
+    'snowp': 'U93KTPQP6',
```

Comment:
I think it would make sense to add buildbreaker, jpsim, and Augustyniak here - this is just for review reminders, right?

Reply:
cc @buildbreaker @jpsim @Augustyniak so when you eventually get slack pings from the "PR reminders" app, you know where they came from :-)